### PR TITLE
Update OGC Sample

### PR DESCRIPTION
**Pull request by nanaeaubry**
Update to use new map methods, folder classes, and layers module

### CODE REVIEW

File: samples/05_content_publishers/publish_ogc_services.ipynb [hunk 1]
```diff
@@ -32,65 +32,12 @@
    "outputs": [],
```
Comment (cariashuang0417) [inline samples/05_content_publishers/publish_ogc_services.ipynb:32]: <pre class="ql-syntax" spellcheck="false">Line #3.    from arcgis.mapping import WebMap
</pre><p>This import can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (cariashuang0417) [inline samples/05_content_publishers/publish_ogc_services.ipynb:32]: <pre class="ql-syntax" spellcheck="false">Line #11.    wms_item = root_folder.add(item_properties=dictItemData)
</pre><p><code>.result()</code> for the Folder.add() job result </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (cariashuang0417) [inline samples/05_content_publishers/publish_ogc_services.ipynb:32]: <pre class="ql-syntax" spellcheck="false">Line #16.    wfs_item = root_folder.add(item_properties=dictItemData)
</pre><p><code>.result()</code> for the Folder.add() result </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (cariashuang0417) [inline samples/05_content_publishers/publish_ogc_services.ipynb:32]: <pre class="ql-syntax" spellcheck="false">Line #8.               )
</pre><p>Same here - <code>.result()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

**Review by cariashuang0417 — CHANGES_REQUESTED** [review]
Some Folder.add are missing `.result()` - which I mentioned in the nbreviewer. Overall looks good to me 😃

**Review by cariashuang0417 — APPROVED** [review]

**Review by jyaistMap — APPROVED** [review]
* submitted #2156 and merged to update description

**Pull request merged**